packages/dx-react-grid-material-ui/src/templates/table-detail-toggle-cell.test.jsx

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 import { mount } from 'enzyme';
+import { Table } from 'material-ui';
```

Comment:
Why do you render cell inside table? Look at the `table-cell.test.jsx` file in the same folder.